### PR TITLE
[11.0][ADD] website_img_dimensions

### PR DESCRIPTION
**Pull request by tarteo**
Simple module that shows the width and height (including original width and height) of images on the website when hovered upon.

Can e.g. be useful when you're optimizing your website.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please avoid plurals on the module name. An screenshot would be good to have it.

**Comment by tarteo** [comment]
@pedrobaeza Thanks for your review. I've changed the name to website_img_dimension. And added a screenshot to the rst.

File: website_img_dimension/static/src/js/website_img_dimension.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+odoo.define('website_img_dimension', function(require) {
```
Comment (simahawk) [inline website_img_dimension/static/src/js/website_img_dimension.js]: website_img_dimension.tooltip

File: website_img_dimension/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Simple module that shows the width and height (including original width and height) of images on the website
```
Comment (simahawk) [inline website_img_dimension/readme/DESCRIPTION.rst]: can you elaborate on the use case?
AFAIU it works only when the editor is loaded: you might want to mention this ;)

File: website_img_dimension/static/src/js/website_img_dimension.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    "use strict";
+    var Class = require('web.Class');
+    var web_editor_base = require('web_editor.base');
+
+    var TooltipManager = Class.extend({
+        init: function() {
+            var self = this;
+            web_editor_base.ready().done(function() {
```
Comment (simahawk) [inline website_img_dimension/static/src/js/website_img_dimension.js]: shouldn't you wrap everything inside this?

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by tarteo** [comment]
@simahawk Thanks for the review :)! I improved the description and changed the name of the js module.

File: website_img_dimension/static/src/js/website_img_dimension.js (hunk 3 above)
Comment (tarteo) [reply website_img_dimension/static/src/js/website_img_dimension.js]: If I wrap the thing in it I wouldn't be able to expose it.

**Pull request closed without merging**